.run/EtlDataPipelineApplication-RDS.run.xml

```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="EtlDataPipelineApplication-RDS" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <envs>
+      <env name="DA_LOG_PATH" value="logs" />
+      <env name="DB_ODSE" value="nbs_odse" />
+      <env name="DB_PASSWORD" value="" />
+      <env name="DB_URL" value="cdc-nbs-legacy-rds-mssql.czya31goozkz.us-east-1.rds.amazonaws.com" />
```

Comment:
Would having the endpoint available in the xml raise any security concerns?

Reply:
Good call. I hope not. The endpoint sits behind a private VPC which means you will require a VPN connection to access.